core/templates/pages/topic-editor-page/modal-templates/rearrange-skills-in-subtopics-modal.component.html

```diff
@@ -0,0 +1,231 @@
+        <mat-card class="subtopic item-container e2e-test-subtopic-column">
+          <div class="subtopic-header"
+               (click)="editNameOfSubtopicWithId(subtopic.getId())">
+            <strong [ngbTooltip]="subtopic.getTitle()"
+                    placement="bottom"
+                    *ngIf="subtopic?.getTitle()?.length < 25">
+              {{ subtopic.getTitle() }}
+            </strong>
+            <strong [ngbTooltip]="subtopic?.getTitle()"
+                    placement="bottom"
+                    *ngIf="subtopic?.getTitle().length >= 25">
+              {{ subtopic?.getTitle().substring(0,15) }}...
+            </strong>
+            <div class="number-skills-container">
+              <span *ngIf="subtopic.getSkillSummaries().length === 1"
+                    class="number-skills-text">
+                {{ subtopic?.getSkillSummaries().length }} Skill
+              </span>
+              <span *ngIf="!(subtopic?.getSkillSummaries().length === 1)"
+                    class="number-skills-text">
+                {{ subtopic?.getSkillSummaries().length }} Skills
+              </span>
+            </div>
+          </div>
+          <div *ngIf="subtopic.getId() === selectedSubtopicId">
+            <input type="text" [(ngModel)]="editableName">
+            <button class="btn btn-secondary"
+                    (click)="editNameOfSubtopicWithId(null)">
+              Cancel
+            </button>
+            <button class="btn btn-primary"
+                    (click)="updateSubtopicTitle(subtopic.getId())">
+              Save
+            </button>
+          </div>
+          <div class="subtopic-container outside-container"
+               cdkDropList
+               [cdkDropListData]="subtopic?.getSkillSummaries()"
+               (cdkDropListDropped)="onMoveSkillEnd($event, subtopic.getId())">
+            <div class="skill-id"
+                      *ngFor="let skillSummary of subtopic.getSkillSummaries()"
+                      cdkDrag
+                      (cdkDragStarted)="onMoveSkillStart(subtopic.getId(), skillSummary)">
+              <div class="skill-card">
+                <a [href]="getSkillEditorUrl(skillSummary?.getId())"
+                   class="skill-name e2e-test-subtopic-skill-description"
+                   target="_blank"
+                   rel="noopener">
+                  {{ skillSummary.getDescription() }}
+                </a>
+              </div>
+            </div>
+          </div>
+        </mat-card>
+      </div>
+    </div>
+  </div>
+  <div class="modal-footer">
+    <button class="btn btn-secondary"
+            (click)="cancel()">
+      Cancel
+    </button>
+    <button class="btn btn-success e2e-test-save-rearrange-skills"
+            (click)="confirm()">
+      Save
+    </button>
+  </div>
+</div>
+<style>
+  .modal-dialog {
+    max-width: inherit;
+    width: 80vw;
+  }
+  .modal-body {
+    height: 55vh;
+    padding: 40px 0 0 0;
+  }
+  a {
+    color: #666;
+  }
```

Comment:
Can the css class be made more specific and descriptive to ensure it does not cause issues with other `a` html tags on the page? ( or by any chance `a` tags of other pages)